--- origsrc/libepoxy-1.2/configure.ac	2014-05-13 19:22:08.000000000 -0500
+++ src/libepoxy-1.2/configure.ac	2015-03-25 14:13:03.832540000 -0500
@@ -70,12 +70,17 @@ case $host_os in
         # opengl32.dll.  But that's the only GL provider, anyway.
         EPOXY_LINK_LIBS="-lopengl32"
 
+        case $build_os in
+        cygwin*|mingw*) ;;
+        *)
         # Testing our built windows binaries requires that they be run
         # under wine.  Yeah, we should be nice and autodetect, but
         # there's lots of missing autodetection for the testsuite
         # (like checking for EGL and GLX libs in non-windows.).
         AC_SUBST([LOG_COMPILER], [wine])
         ;;
+        esac
+        ;;
     darwin*)
         build_egl=no
         build_glx=yes
